### PR TITLE
Add the Queen's platinum jubilee

### PR DESCRIPTION
**Pull request by hlascelles**
This is for the wonderous news of the Queen's platinum jubilee: https://www.bbc.co.uk/news/uk-54911550

This has been confirmed by the UK Gov site: https://www.gov.uk/bank-holidays

image

Unfortunately I am struggling to make this change.

I have added what I think are two tests for the new dates. NB, another date (the late May bank holiday) has been delayed to form one of these new dates. 

So, using TDD, I would expect this change as-is to fail twice with the two new tests. Ultimately I would hope to make it fail at least 3 times (including the move of the previous holiday) before I even started on adding the new definitions.

But running the following, everything passes fine:

```bash
git clone git@github.com:holidays/holidays.git
cd holidays
bundle install
BRANCH=add-queens-jubilee USER=hlascelles make point-to-defs-branch
make test
# Passes!
```

To the best of my knowledge that is what I have been asked to check. I must confess I find the submission process hard to follow. Why is everything passing?

More generally, the barrier to entry for submissions is quite high. I know this has been discussed before, and my 2 cents is that it could be easier...

Any guidance appreciated!

### CODE REVIEW

**Comment by ttwo32** [comment]
@hlascelles Thank you for contribution.
And I'm sorry for my lazy response...

Maybe , a command that is 'make generate' is missing.
('generate' will converte yaml to ruby-codes.)
When I tried it ,two tests were failed.
(Tests in other regions also failed, but I'm fixing it.)

Could you try this?
Thanks.

**Comment by WorkSutton** [comment]
@hlascelles Can this be closed now as it seems to be a duplicate of https://github.com/holidays/definitions/pull/235 which was merged 2 hours ago which was I believe also your PR. Nice work by the way and thank you

**Comment by frankieroberto** [comment]
> @hlascelles Can this be closed now as it seems to be a duplicate of #235 which was merged 2 hours ago which was I believe also your PR. Nice work by the way and thank you

#235 is a separate bank holiday (Queen’s funeral) and this one was for the Queen’s jubilee - but it's a duplicate of #207 and can be closed.

**Comment by hlascelles** [comment]
No problem, glad they're all in!
